docs/src/main/paradox/routing-dsl/testkit.md

```diff
@@ -222,6 +222,23 @@ Java
 
 Remember to configure the timeout using `dilated` if you want to account for slow test systems.
 
+## Integration Testing Routes
+
+Use `~!>` to test a route running in full HTTP server mode:
+```
+REQUEST ~!> ROUTE ~> check {
+  ASSERTIONS
+}
+```
+
+Certain routes can only be tested with `~!>`, for example routes that use
+the `withRequestTimeout` directive.
+
+@@@ note
+Using `~!>` adds considerable extra overhead since each test will start
+a server and bind to a port so use it only when necessary.
```

Comment:
>  Using `~!>` adds considerable extra overhead since each test will start a server and bind to a port so use it only when necessary.

Would it be possible to let the request be routed to a already running server and not start/stop a new server for each test. Objective being that my server is already running in stage environment and we would like to implement some integration test using the test routing DSL. From my quick read of this PR `~!>` looks just the right solution ... just that instead of the test framework starting a new server it can connect to an existing server

Reply:
I think using 'regular' requests might be easier in that case.

What part of the test routing DSL is hard to recreate with checks on the 'regular' response?